Nos/Service/NostrEntity/TLVEntity.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+enum TLVEntity {
+    case special(value: String)
+    case relay(value: String)
+    case author(value: String)
+    case kind(value: UInt32)
+    case unknown
+
+    static func decodeEntities(data: Data) -> [TLVEntity] {
+        guard let converted = try? data.base8FromBase5() else {
+            return []
+        }
+
+        var result: [TLVEntity] = []
+
+        var offset = 0
+        while offset + 1 < converted.count {
+            let rawType = converted[offset]
+            let type = TLVType(rawValue: rawType)
+            let length = Int(converted[offset + 1])
+            let value = converted.subdata(in: offset + 2 ..< offset + 2 + length)
+
+            let entity: TLVEntity
+            switch type {
+            case .special:
+                let valueString = SHA256Key.decode(base8: value)
+                entity = .special(value: valueString)
+            case .relay:
+                let valueString = String(data: value, encoding: .ascii)
+                entity = .relay(value: valueString ?? "") // TODO: handle error
```

Comment:
Should we return nil here?

Reply:
Yeah, we should probably just skip over this one and continue processing. I'll do that.

Reply:
✅